Cargo.toml

```diff
@@ -50,6 +50,7 @@ tracing-subscriber = { version = "0.3.19", default-features = false, features =
     "json",
     "env-filter",
 ] }
+subprocess = { version = "0.2.9" }
```

Comment:
This was last released 3 years ago. I'd prefer if we could just rawdog the std library for process management, I find it sufficient.